docs/docs/addons-perf.md

```diff
@@ -56,6 +60,36 @@ The following methods use the measurements returned by [`Perf.getLastMeasurement
 
 * * *
 
+## Example Usage
+
+We will take the [simple example]() of a form which can be used to submit comments and then listing all comments. The example code is availabe on [github](https://github.com/dhyey35/react-example-for-performance/blob/master/public/scripts/example.js).
+
+```javascript
+import { Component } from 'react'
+import Perf from 'react-addons-perf'
+
+const Comment = (props) => {
+  return (
+     <div className="comment">
```

Comment:
Minor: Looks like you have some inconsistent spacing here. (sometimes 2, sometimes 3)...